src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -144,8 +144,12 @@ public void updateTeamsForPrimaryTopic(String caseUUIDString, String stageUUIDSt
 
         Map<String, String> teamsForTopic = new HashMap<>();
         TeamDto teamDto = infoClient.getTeamForTopicAndStage(caseUUID, topicUUID, stageType);
-        teamsForTopic.put(teamNameKey, teamDto.getUuid().toString());
-        teamsForTopic.put(teamUUIDKey, teamDto.getDisplayName());
+        if (teamDto.isActive()) {
```

Comment:
Is there any chance that 'teamDto' will be null at this point? (e.g. if topic has no assigned team)